lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -84,13 +85,15 @@ sub _get_ptero_shortcut_method {
 }
 
 sub _get_ptero_execute_method {
-    require Ptero::Builder::ShellCommand;
+    require Ptero::Builder::Job;
+
 
     my $self = shift;
     my $log_dir = shift;
-    # XXX This should use the LSF service, or be configuration based
-    return Ptero::Builder::ShellCommand->new(
+    return Ptero::Builder::Job->new(
         name => 'execute',
+        # XXX This should use the LSF service, or be configuration based
```

Comment:
There will be a different environment variable that points to the lsf service url and the resource string (if it exists) on the command should be included in the parameters.